src/sidebar/cross-origin-rpc.js

```diff
@@ -38,28 +53,25 @@ function start(store, settings, $window) {
 
   /** Return a JSON-RPC response to the given JSON-RPC request object. */
   function jsonRpcResponse(request) {
-    // The set of methods that clients can call.
-    let methods = {
-      searchUris: store.searchUris,
-    };
-
-    let method = methods[request.method];
-
-    let response = {
-      jsonrpc: '2.0',
-      id: request.id,
-    };
+    // Find the mapped method requested
```

Comment:
Ditto with this comment, and I the `allowedMethods` naming would help here, too, methinks.